website/docs/podman/accessing-podman-from-another-wsl-instance.md

```diff
@@ -0,0 +1,193 @@
+     ```
+
+   - Rootless Podman:
+
+     ```shell-session
+     $ grep :1000: /etc/group
+     ```
+
+   </div>
+   </details>
+
+## Assigning the Podman machine destination
+
+Assigning the Podman machine destination for a Podman client
+
+- Set an alias to use the remote Podman without modifying how you start the Podman CLI:
+
+  ```shell-session
+  $ alias podman='podman --remote --uri="unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-root.sock"'
+  ```
+
+  <details>
+
+  <summary>
+  Finding the Podman machine remote destination on a custom Podman machine
+  </summary>
+  <div>
+
+  A custom Podman machine might have a different name.
+  You need to find your Podman machine remote destination.
+
+  1. Start a session in your WSL distribution:
+
+     ```shell-session
+     > wsl
+     ```
+
+  1. The Podman machine shares sockets in a `/mnt/wsl/podman-sockets/` subdirectory named after the Podman machine name.
+     List the available sockets, and identify your remote Podman machine connection in the list.
+
+     ```shell-session
+     $ ls --numeric--uid-gid --recursive /mnt/wsl/podman-sockets/
+     ```
+
+  1. To define the Podman machine remote destination, prepend the found path with `uri://`:
+
+     For the default Podman machine:
+
+     - Rootful Podman: `unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-root.sock`
+     - Rootless Podman: `unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-user.sock`
+
+  </div>
+  </details>
+
+  <details>
+  <summary>
+  Configuring a Podman v4 client on a custom WSL distribution
+  </summary>
+
+  With Podman v4, you can use the Podman configuration file.
+  Podman does not need the `--remote` flag when you have set a remote service destination.
+
+  To assign the remote service destination, in your WSL distribution, configure the `[engine.service_destinations]` table in the Podman configuration file.
+
+  1. Create the Podman configuration file if it does not exist.
+
+     ```shell-session
+     $ mkdir -p ~/.config/containers/containers.conf.d
+     $ touch ~/.config/containers/containers.conf.d/destinations.conf
+     ```
+
+  1. Add these lines in the `~/.config/containers/containers.conf.d/destinations.conf` file:
+
+     - Default Rootful Podman machine:
+
+       ```toml
+       [engine.service_destinations]
+       [engine.service_destinations.production]
+       uri="unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-root.sock"
+       ```
+
+     - Default Rootless Podman machine:
+
+       ```toml
+       [engine.service_destinations]
+       [engine.service_destinations.production]
+       uri="unix:///mnt/wsl/podman-sockets/podman-machine-default/podman-user.sock"
+       ```
+
+  </details>
+
+## Testing the connection to the remote Podman machine
+
+On your WSL distribution, the Podman CLI communicates with your Podman machine.
+
+- List running containers to verify the connection:
+
+  ```shell-session
+  $ podman ps -a
```

Comment:
I'm not sure that ensure Podman in the user's WSL environment will communicate to the Podman used by Podman Desktop.

I'll probably start by creating a container in podman desktop and then checking that the container is visible in the environment the user wanted to configure...

Reply:
Good idea.